cluster-up/cluster/kind/common.sh

```diff
@@ -141,6 +141,28 @@ function _fix_node_labels() {
     done
 }
 
+function _get_cri_bridge_mtu() {
+  docker network inspect bridge  | grep mtu | awk '{print $2}' | sed "s/\"//g"
+}
+
+function _patch_calico_manifest_mtu() {
+  local -r calico_manifest="$1"
+  local -r mode="$2"
+  local -r cri_mtu="$3"
+
+  local -r default_calico_mtu="1440"
+
+  if [ $mode == "ipip" ]; then
+    overhead=$((20))
+  elif [ $mode == "vxlan" ];then
+    overhead=$((50))
```

Comment:
This is dead code at the moment, there is no way to test this and therefore I prefer it out until we really need it.
I would prefer you read the mode from the manifest, confirm it is ipip and then patch.

Reply:
Done